### ISSUE TITLE
Limiting time selections for rooms by day of the week

### ISSUE THREAD

**Issue by TLamanna** [ldjamison/koha-plugin-room-reservations#16]
I can bulk remove time slots w/ jquery, but is there an option to allow time slot selections by the date they picked? We have different hours depending on the day of the week and want to make sure our room reservations reflect this.

**Comment by ldjamison** [comment]
Hi! So you are referring to almost like a "recurring blackout"? (see the wiki for more on what I mean by blackouts).

**Comment by TLamanna** [comment]
Exactly, I know I can use the blackouts to cover days we're closed, but for regular variable hours it would be nice to be able to limit slots by date selected. For some reason to jquery change() function is working, but I know I can .remove() a bunch of them (but can't make it selection dependent. I'm going to keep going on my workaround and I'll send you what I've done if you like.

**Comment by ldjamison** [comment]
I think that certainly makes sense. I'll work on adding this in since it seems to be just a slight modification to the blackout feature. What version of the plugin are you using?

**Comment by TLamanna** [comment]
1.1.12

**Comment by TLamanna** [comment]
The blackout function works, but I think it would be more user friendly to have the blackout times/days be restricted on the drop down rather than just not showing an available room. Sorry for being such a pest.

**Comment by ldjamison** [comment]
Not a pest at all! I've been busy so I'm just getting back to enhancement work. I think this issue pairs in with Issue #11 as I think it functionally makes the most sense. I need to do a bit of prerequisite investigation into how the Koha calendar saves/populates but I feel that would be the best direction to take this. Let me know what you think
